specification/columns/commitmentdiscountcategory.md

```diff
@@ -2,6 +2,19 @@
 
 Commitment Discount Category indicates whether the [*commitment discount*](#glossary:commitment-discount) identified in the CommitmentDiscountId column is based on usage quantity or cost (aka "spend"). The CommitmentDiscountCategory column is only applicable to *commitment discounts* and not [*negotiated discounts*](#glossary:negotiated-discount).
 
+---
+The CommitmentDiscountCategory column adheres to the following requirements:
+
+* CommitmentDiscountCategory MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) when the provider supports *commitment discounts*.
+* If present, the column adheres to the following additional requirements:
+  * CommitmentDiscountCategory MUST be of type String.
+  * CommitmentDiscountCategory MUST conform to [String Handling](#stringhandling) requirements.
+  * CommitmentDiscountCategory MUST be null if [CommitmentDiscountId](#commitmentdiscountid) is null.
+  * If CommitmentDiscountId is not null, the following applies:
```

Comment:
```suggestion
  * If CommitmentDiscountId is not null, the column adheres to the following additional requirements:
```

Reply:
suggested change applied across the board